Sources/MySQL/Database.swift

```diff
@@ -64,6 +67,21 @@ public final class Database {
         self.socket = socket
         self.flag = flag
         self.encoding = encoding
+        
+        // Clamp connection pool size to range of 1-32
+        let poolSize = pool < 1 ? 1 :
+            pool > 32 ? 32 :
+            pool
+        
+        self.connectionPool = [Connection]()
+        for _ in 0..<poolSize {
+            do {
+                let conn = try self.makeConnection()
```

Comment:
Could this make the app sluggish to start? Would probably be better to create new connections as we need them then store them when done.